File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main4Activity.java
package com.alejouribesanchez.class_4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class Main4Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main4);

        Button button =  (Button)findViewById(R.id.boton_iniciar_sesion);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(Main4Activity.this, Main5Activity.class);
                startActivity(intent);
            }
        });

    }
}

File: class_4/class_4/app/src/main/java/com/alejouribesanchez/class_4/Main4Activity.kt
package com.alejouribesanchez.class_4

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button

class Main4Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main4)

        val button = findViewById(R.id.boton_iniciar_sesion) as Button
        button.setOnClickListener {
            val intent = Intent(this@Main4Activity, Main5Activity::class.java)
            startActivity(intent)
        }

    }
}
